logstash_formatter/__init__.py

```diff
@@ -84,32 +84,28 @@ def format(self, record):
         except (KeyError, IndexError):
             pass
 
-        if 'msg' in fields:
-            fields.pop('msg')
-
-        if 'exc_info' in fields:
-            if fields['exc_info']:
-                formatted = tb.format_exception(*fields['exc_info'])
-                fields['exception'] = formatted
-            fields.pop('exc_info')
+        if fields['exc_info']:
```

Comment:
do you still want to check for `exc_info` in the fields dictionary? If this doesn't exist, it'll throw a key error